src/views/Login/Login.jsx

```diff
@@ -199,8 +217,9 @@ export class Login extends React.Component {
             </button>
             <a
               id="forgot-password"
-              href="javascript:;;"
-              title={i18n.t('login.basic.forgot.title')}>
+              href="#TODO"
```

Comment:
What's this? Do we actually display any sort of forgot password flow?

Reply:
No, we have no forgot password flow. I added the `#TODO` to make it more obvious that this link doesn't do anything but I'm not sure if there is a plan for it or if we should just remove it altogether.

Reply:
I vote we delete it. Dead code tells no tales.